health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateCampaignComponents/NewUploadData.js

```
@@ -0,0 +1,1326 @@
+          const { data: { fileStoreIds: fileUrl } = {} } = await Digit.UploadServices.Filefetch(filesArray, tenantId);
+          const fileData = fileUrl?.map((i) => {
+            const urlParts = i?.url?.split("/");
+            // const fileName = urlParts[urlParts?.length - 1]?.split("?")?.[0];
+            const fileName = type === "boundary" ? "Target Template" : type === "facility" ? "Facility Template" : "User Template";
+            return {
+              ...i,
+              filename: fileName,
+            };
+          });
+
+          if (fileData && fileData?.[0]?.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)